### PR TITLE
Add source to NEF

### PR DESCRIPTION
**Pull request by shargon**

### CODE REVIEW

**Comment by lllwvlvwlll** [comment]
@shargon do you have some examples of how this can be used?  Is there a standard format (like absolute path to the entrypoint) that this is being implemented alongside?

**Review by devhawk — CHANGES_REQUESTED** [review]
This information is already in the debug info. https://github.com/devhawk/proposals/blob/devhawk/debug-info/nep-19.md#documents

I don't see the value of putting it in the NEF file

**Comment by shargon** [comment]
> I don't see the value of putting it in the NEF file

It's already in the nef file

**Comment by shargon** [comment]
@lllwvlvwlll it was introduced in https://github.com/neo-project/neo/pull/2605 , I think that it must be an absolute url.

**Comment by devhawk** [comment]
> It's already in the nef file https://github.com/neo-project/neo/blob/master/src/neo/SmartContract/NefFile.cs#L57

Then I agree it should be in the NEP

**Review by devhawk — APPROVED** [review]

File: nep-16.mediawiki [hunk 1]
```diff
@@ -30,6 +30,7 @@ nef_file
 {
   4-byte  magic;
   64-byte compiler;
+  var-byte source;
   2-byte reserve;
```
Comment (erikzhang) [inline nep-16.mediawiki:34]: Then it becomes 1 byte.

Comment (shargon) [reply nep-16.mediawiki:34]: At least one, but it can be more

**Review by erikzhang — APPROVED** [review]

**Pull request merged**